react/components/Input/index.js

```diff
@@ -208,7 +223,7 @@ class Input extends Component {
           )}
           <input
             {...dataAttrs}
-            ref={this.props.forwardedRef}
+            ref={this.props.forwardedRef || this.ref}
```

Comment:
can you do something like
```suggestion
            ref={node => {
              if (typeof this.props.forwardedRef === 'function') {
                this.props.forwardedRef(node)
              } else {
                this.props.forwardedRef.current = node
              }
              this.ref.current = node
            }}
```

and change the `handleAutoFocus` to only use the `this.ref`

Reply:
I think this could be done in another PR

Reply:
fair enough

Reply:
What do you guys think about this implementation: https://codesandbox.io/s/merging-refs-v2sry ?

Reply:
perfect 👌

Reply:
Opened #1110 for that, @lucasecdb & @Klynger